categories/models.py

```diff
@@ -10,10 +10,24 @@ class Category(TranslatableModel):
         description=models.TextField(verbose_name=_("description"), blank=True),
     )
 
+    principle = models.BooleanField(default=False)
+
     class Meta:
         verbose_name = _("category")
         verbose_name_plural = _("categories")
-        ordering = ("id",)
+        ordering = (
+            "-principle",
+            "translations__name",
+        )
 
     def __str__(self):
-        return self.safe_translation_getter("name", super().__str__())
+        return f'{self.safe_translation_getter("name", super().__str__())}\
+                  is principle: {self.principle}'
+
+
+class CategoriesTable(models.Model):
```

Comment:
IMO having the word "table" in a model's name leads to confusion and unclarity. would a better name be something like "FeatureCategory"? it is a common practice in Django development to combine the name of the two models for the thorugh model between them in m2m relations. or if there is an even more descriptive other word, it would be even better, but I cannot really come up with anything right here and now 🤔